{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 141,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import requests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "metadata": {},
   "outputs": [],
   "source": [
    "# CMU's Covidcast API: https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html\n",
    "# Citation: David C. Farrow, Logan C. Brooks, Aaron Rumack, Ryan J. Tibshirani, Roni Rosenfeld (2015). Delphi Epidata API. https://github.com/cmu-delphi/delphi-epidata\n",
    "import covidcast"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "metadata": {},
   "outputs": [],
   "source": [
    "import datetime as dt\n",
    "from datetime import date"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "metadata": {},
   "outputs": [],
   "source": [
    "yr = 2020\n",
    "strmon = 2\n",
    "strdt = 1\n",
    "endmon = 11\n",
    "enddt = 18"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## pulling data by state for the following signals\n",
    "\n",
    "safegraph //\n",
    "    completely_home_prop /\n",
    "    full_time_work_prop /\n",
    "    part_time_work_prop /\n",
    "    \n",
    "ght //\n",
    "    smoothed_search /\n",
    "    \n",
    "NONE: doctor_visits  //\n",
    "    smoothed_adj_cli /\n",
    "    \n",
    "fb-survey  //\n",
    "    smoothed_cli /\n",
    "    smoothed_ili /\n",
    "    \n",
    "quidel //\n",
    "    covid_ag_smoothed_pct_positive /\n",
    "    \n",
    "hospital-admissions //\n",
    "    smoothed_covid19 /\n",
    "\n",
    "jhu-csse //\n",
    "    confirmed_incidence_prop /\n",
    "    deaths_incidence_prop /"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201116: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201117: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201118: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    }
   ],
   "source": [
    "move_home = covidcast.signal(\"safegraph\", \"completely_home_prop\", \n",
    "                         date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "           `             geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "move_fullw = covidcast.signal(\"safegraph\", \"full_time_work_prop\", \n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "move_partw = covidcast.signal(\"safegraph\", \"part_time_work_prop\", \n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "doc_visit = covidcast.signal(\"doctor-visits\", \"smoothed_adj_cli\", \n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "survey_c = covidcast.signal(\"fb-survey\", \"smoothed_ili\", \n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200201: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200202: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200203: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200204: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200205: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200206: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200207: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200208: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200209: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200210: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200211: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200212: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200213: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200214: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200215: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200216: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200217: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200218: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200219: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200220: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200221: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200222: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200223: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200224: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200225: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200226: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200227: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200228: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200229: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200301: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200302: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200303: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200304: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200305: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200306: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200307: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200308: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200309: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200310: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200311: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200312: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200313: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200314: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200315: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200316: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200317: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200318: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200319: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200320: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200321: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200322: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200323: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200324: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200325: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200326: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200327: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200328: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200329: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200330: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200331: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200401: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200402: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200403: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200404: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200405: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    }
   ],
   "source": [
    "survey_i = covidcast.signal(\"fb-survey\", \"smoothed_cli\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "hv = covidcast.signal(\"hospital-admissions\", \"smoothed_covid19_from_claims\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200406: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200407: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200408: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200409: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200410: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200411: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200412: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200413: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200414: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200415: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200416: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200417: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200418: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200419: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200420: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200421: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200422: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200423: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200424: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200425: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200426: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200427: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200428: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200429: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200430: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200501: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200502: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200503: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200504: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200505: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200506: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200507: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200508: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200509: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200510: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200511: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200512: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200513: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200514: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200515: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200516: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200517: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200518: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200519: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200520: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200521: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200522: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200523: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200524: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200525: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201112: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    }
   ],
   "source": [
    "c_tests = covidcast.signal(\"quidel\", \"covid_ag_smoothed_pct_positive\",\n",
    "                          date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200817: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200821: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200919: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201103: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201104: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201105: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201106: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201107: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201108: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    }
   ],
   "source": [
    "search = covidcast.signal(\"ght\", \"smoothed_search\",\n",
    "                         date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "jhu_case = covidcast.signal(\"jhu-csse\", \"confirmed_incidence_prop\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "jhu_deaths_100 = covidcast.signal(\"jhu-csse\", \"deaths_incidence_prop\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"state\")\n",
    "jhu_deaths_abs = covidcast.signal(\"jhu_case\", \"confirmed_incidence_num\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type='state')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "move_home.rename(columns={'value': 'move_home'}, inplace=True)\n",
    "move_fullw.rename(columns={'value': 'move_fullw'}, inplace=True)\n",
    "move_partw.rename(columns={'value': 'move_partw'}, inplace=True)\n",
    "search.rename(columns={'value': 'search'}, inplace=True)\n",
    "doc_visit.rename(columns={'value': 'doc_visit'}, inplace=True)\n",
    "survey_c.rename(columns={'value': 'survey_c'}, inplace=True)\n",
    "survey_i.rename(columns={'value': 'survey_i'}, inplace=True)\n",
    "c_tests.rename(columns={'value': 'c_tests'}, inplace=True)\n",
    "hv.rename(columns={'value': 'hv'}, inplace=True)\n",
    "jhu_case.rename(columns={'value': 'jhu_case'}, inplace=True)\n",
    "jhu_deaths.rename(columns={'value': 'jhu_d'}, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub1 = move_home[[\"geo_value\", \"time_value\", \"move_home\"]]\n",
    "sub2 = move_fullw[[\"geo_value\", \"time_value\", \"move_fullw\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub3 = move_partw[[\"geo_value\", \"time_value\", \"move_partw\"]]\n",
    "sub4 = search[[\"geo_value\", \"time_value\", \"search\"]]\n",
    "sub5 = doc_visit[[\"geo_value\", \"time_value\", \"doc_visit\"]]\n",
    "sub6 = survey_c[[\"geo_value\", \"time_value\", \"survey_c\"]]\n",
    "sub7 = survey_i[[\"geo_value\", \"time_value\", \"survey_i\"]]\n",
    "sub8 = c_tests[[\"geo_value\", \"time_value\", \"c_tests\"]]\n",
    "sub9 = hv[[\"geo_value\", \"time_value\", \"hv\"]]\n",
    "sub10 = jhu_case[[\"geo_value\", \"time_value\", \"jhu_case\"]]\n",
    "sub11 = jhu_deaths[[\"geo_value\", \"time_value\", \"jhu_d\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "move_full = sub1.merge(sub2, how='outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "move_full = move_full.merge(sub3, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub45 = sub4.merge(sub5, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub67 = sub6.merge(sub7, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub89 = sub8.merge(sub9, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub1011 = sub10.merge(sub11, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub47 = sub45.merge(sub67, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub811 = sub89.merge(sub1011, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub411 = sub47.merge(sub811, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_symp = sub411.merge(move_full, how = 'outer', left_on = [\"geo_value\", \"time_value\"], right_on = [\"geo_value\", \"time_value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_symp.fillna(0, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>geo_value</th>\n",
       "      <th>time_value</th>\n",
       "      <th>search</th>\n",
       "      <th>doc_visit</th>\n",
       "      <th>survey_c</th>\n",
       "      <th>survey_i</th>\n",
       "      <th>c_tests</th>\n",
       "      <th>hv</th>\n",
       "      <th>jhu_case</th>\n",
       "      <th>jhu_d</th>\n",
       "      <th>move_home</th>\n",
       "      <th>move_fullw</th>\n",
       "      <th>move_partw</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>15988</th>\n",
       "      <td>mp</td>\n",
       "      <td>2020-10-25</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.846154</td>\n",
       "      <td>0.076923</td>\n",
       "      <td>0.076923</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15989</th>\n",
       "      <td>vi</td>\n",
       "      <td>2020-10-25</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.338626</td>\n",
       "      <td>0.067137</td>\n",
       "      <td>0.083679</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      geo_value time_value  search  doc_visit  survey_c  survey_i  c_tests  \\\n",
       "15988        mp 2020-10-25     0.0        0.0       0.0       0.0      0.0   \n",
       "15989        vi 2020-10-25     0.0        0.0       0.0       0.0      0.0   \n",
       "\n",
       "        hv  jhu_case  jhu_d  move_home  move_fullw  move_partw  \n",
       "15988  0.0       0.0    0.0   0.846154    0.076923    0.076923  \n",
       "15989  0.0       0.0    0.0   0.338626    0.067137    0.083679  "
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_symp.tail(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15990, 13)"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_symp.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "statell = pd.read_csv('state_ll.csv')\n",
    "statell = statell.drop(columns = [\"state\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_symp = full_symp.merge(statell, how = 'outer', left_on='geo_value', right_on='state_lower')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_symp.to_csv(\"data_state.covidcast_11.16.20.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>geo_value</th>\n",
       "      <th>time_value</th>\n",
       "      <th>search</th>\n",
       "      <th>doc_visit</th>\n",
       "      <th>survey_c</th>\n",
       "      <th>survey_i</th>\n",
       "      <th>c_tests</th>\n",
       "      <th>hv</th>\n",
       "      <th>jhu_case</th>\n",
       "      <th>jhu_d</th>\n",
       "      <th>move_home</th>\n",
       "      <th>move_fullw</th>\n",
       "      <th>move_partw</th>\n",
       "      <th>latitude</th>\n",
       "      <th>longitude</th>\n",
       "      <th>name</th>\n",
       "      <th>state_lower</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ak</td>\n",
       "      <td>2020-02-01</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.133990</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.309489</td>\n",
       "      <td>0.032409</td>\n",
       "      <td>0.057572</td>\n",
       "      <td>63.588753</td>\n",
       "      <td>-154.493062</td>\n",
       "      <td>Alaska</td>\n",
       "      <td>ak</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ak</td>\n",
       "      <td>2020-02-02</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.189365</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.346763</td>\n",
       "      <td>0.029233</td>\n",
       "      <td>0.047159</td>\n",
       "      <td>63.588753</td>\n",
       "      <td>-154.493062</td>\n",
       "      <td>Alaska</td>\n",
       "      <td>ak</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ak</td>\n",
       "      <td>2020-02-03</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.227790</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.243155</td>\n",
       "      <td>0.044904</td>\n",
       "      <td>0.084309</td>\n",
       "      <td>63.588753</td>\n",
       "      <td>-154.493062</td>\n",
       "      <td>Alaska</td>\n",
       "      <td>ak</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ak</td>\n",
       "      <td>2020-02-04</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.187205</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.240382</td>\n",
       "      <td>0.055521</td>\n",
       "      <td>0.093727</td>\n",
       "      <td>63.588753</td>\n",
       "      <td>-154.493062</td>\n",
       "      <td>Alaska</td>\n",
       "      <td>ak</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ak</td>\n",
       "      <td>2020-02-05</td>\n",
       "      <td>25.435233</td>\n",
       "      <td>0.071758</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.263920</td>\n",
       "      <td>0.063959</td>\n",
       "      <td>0.112363</td>\n",
       "      <td>63.588753</td>\n",
       "      <td>-154.493062</td>\n",
       "      <td>Alaska</td>\n",
       "      <td>ak</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  geo_value time_value     search  doc_visit  survey_c  survey_i  c_tests  \\\n",
       "0        ak 2020-02-01   0.000000   0.133990       0.0       0.0      0.0   \n",
       "1        ak 2020-02-02   0.000000   0.189365       0.0       0.0      0.0   \n",
       "2        ak 2020-02-03   0.000000   0.227790       0.0       0.0      0.0   \n",
       "3        ak 2020-02-04   0.000000   0.187205       0.0       0.0      0.0   \n",
       "4        ak 2020-02-05  25.435233   0.071758       0.0       0.0      0.0   \n",
       "\n",
       "    hv  jhu_case  jhu_d  move_home  move_fullw  move_partw   latitude  \\\n",
       "0  0.0       0.0    0.0   0.309489    0.032409    0.057572  63.588753   \n",
       "1  0.0       0.0    0.0   0.346763    0.029233    0.047159  63.588753   \n",
       "2  0.0       0.0    0.0   0.243155    0.044904    0.084309  63.588753   \n",
       "3  0.0       0.0    0.0   0.240382    0.055521    0.093727  63.588753   \n",
       "4  0.0       0.0    0.0   0.263920    0.063959    0.112363  63.588753   \n",
       "\n",
       "    longitude    name state_lower  \n",
       "0 -154.493062  Alaska          ak  \n",
       "1 -154.493062  Alaska          ak  \n",
       "2 -154.493062  Alaska          ak  \n",
       "3 -154.493062  Alaska          ak  \n",
       "4 -154.493062  Alaska          ak  "
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_symp.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## pulling data by county for the following signals\n",
    "\n",
    "safegraph //\n",
    "    completely_home_prop /\n",
    "    full_time_work_prop /\n",
    "    part_time_work_prop /\n",
    "    \n",
    "doctor_visits  //\n",
    "    smoothed_adj_cli /\n",
    "    \n",
    "fb-survey  //\n",
    "    smoothed_cli /\n",
    "    smoothed_ili /\n",
    "    \n",
    "hospital-admissions //\n",
    "    smoothed_covid19 /\n",
    "\n",
    "jhu-csse //\n",
    "    confirmed_incidence_prop /\n",
    "    deaths_incidence_prop /\n",
    "    confirmed_incidence_num /"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# see 'cfips.csv' for all US counties with lat/long"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "metadata": {},
   "outputs": [],
   "source": [
    "# list of counties in Hawaii\n",
    "hiclist = [\"01001\", \"01003\", \"01005\", \"01007\", \"01009\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [],
   "source": [
    "# list of counties in Florida\n",
    "dfflc = pd.read_csv('fl_counties.csv', encoding='iso-8859-1')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfflc['longitude'] = dfflc['longitude'].str[3:-2]\n",
    "dfflc['longitude'] = '-' + dfflc['longitude'].astype(str)\n",
    "dfflc['latitude'] = dfflc['latitude'].str[1:-2]\n",
    "dfflc = dfflc.drop('ï»¿state', axis=1)`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {},
   "outputs": [],
   "source": [
    "flclist = dfflc['FIPS'].values.tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 145,
   "metadata": {},
   "outputs": [],
   "source": [
    "tampalist = [\"12057\", \"12103\", \"12081\", \"12115\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "metadata": {},
   "outputs": [],
   "source": [
    "# variable used to identify which county fips list to use\n",
    "ctylist = tampalist"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201115: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    }
   ],
   "source": [
    "move_home = covidcast.signal(\"safegraph\", \"completely_home_prop\", \n",
    "                         date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                         geo_type=\"county\", geo_values=ctylist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {},
   "outputs": [],
   "source": [
    "move_fullw = covidcast.signal(\"safegraph\", \"full_time_work_prop\", \n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 149,
   "metadata": {},
   "outputs": [],
   "source": [
    "move_partw = covidcast.signal(\"safegraph\", \"part_time_work_prop\", \n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "metadata": {},
   "outputs": [],
   "source": [
    "doc_visit = covidcast.signal(\"doctor-visits\", \"smoothed_adj_cli\", \n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 151,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200201: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200202: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200203: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200204: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200205: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200206: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200207: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200208: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200209: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200210: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200211: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200212: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200213: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200214: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200215: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200216: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200217: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200218: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200219: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200220: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200221: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200222: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200223: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200224: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200225: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200226: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200227: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200228: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200229: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200301: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200302: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200303: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200304: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200305: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200306: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200307: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200308: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200309: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200310: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200311: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200312: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200313: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200314: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200315: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200316: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200317: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200318: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200319: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200320: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200321: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200322: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200323: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200324: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200325: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200326: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200327: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200328: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200329: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200330: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200331: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200401: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200402: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200403: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200404: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200405: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200406: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    }
   ],
   "source": [
    "survey_c = covidcast.signal(\"fb-survey\", \"smoothed_ili\", \n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "metadata": {},
   "outputs": [],
   "source": [
    "survey_i = covidcast.signal(\"fb-survey\", \"smoothed_cli\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "metadata": {},
   "outputs": [],
   "source": [
    "hv = covidcast.signal(\"hospital-admissions\", \"smoothed_covid19_from_claims\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 154,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200407: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200408: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200409: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200410: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200411: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200412: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200413: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200414: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200415: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200416: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200417: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200418: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200419: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200420: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200421: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200422: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200423: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200424: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200425: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200426: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200427: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200428: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200429: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200430: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200501: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200502: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200503: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200504: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200505: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200506: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200507: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200508: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200509: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200510: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200511: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200512: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200513: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200514: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200515: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200516: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200517: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200518: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200519: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200520: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200521: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200522: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200523: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200524: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200525: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200526: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200527: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200528: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200529: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200530: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200531: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200601: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200602: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200603: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200604: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200605: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200606: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200607: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200608: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200609: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200610: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200611: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200612: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200613: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200614: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200615: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200616: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200617: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200618: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200619: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200620: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200621: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200622: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200623: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200624: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200625: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200626: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200627: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200628: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200629: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200630: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200701: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200702: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200703: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200704: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200705: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200706: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200707: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200708: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200709: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200710: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200711: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200712: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200713: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200714: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200715: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200716: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200717: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200718: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200719: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200720: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200721: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200722: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200723: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200724: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200725: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200726: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200727: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200728: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200729: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200730: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200731: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200801: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200802: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200803: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200804: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200805: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200806: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200807: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200808: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200809: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200810: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200811: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200812: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200813: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200814: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200815: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200816: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200817: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200818: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200819: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200820: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200821: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200822: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200823: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200824: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200825: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200826: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200827: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200828: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200829: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200830: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200831: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200901: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200902: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200903: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200904: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200905: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200906: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200907: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200908: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200909: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200910: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200911: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200912: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200913: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200914: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200915: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200916: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200917: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200918: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200919: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200920: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200921: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200922: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200923: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200924: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200925: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200926: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200927: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200928: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200929: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20200930: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201001: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201002: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201003: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201004: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201005: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201006: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201007: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201008: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201009: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201010: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201011: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201012: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201013: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201014: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201015: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201016: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201017: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201018: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201019: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201020: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201021: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201022: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201023: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201024: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201025: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201026: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201027: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201028: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201029: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201030: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201031: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201101: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201102: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201103: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201104: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201105: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201106: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201107: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201108: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201109: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201110: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201111: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201112: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201113: no results\n",
      "  message=day_data[\"message\"]))\n",
      "/Users/annegibbon/anaconda3/lib/python3.6/site-packages/covidcast/covidcast.py:374: UserWarning: Problem obtaining data on 20201114: no results\n",
      "  message=day_data[\"message\"]))\n"
     ]
    }
   ],
   "source": [
    "jhu_case = covidcast.signal(\"jhu-csse\", \"confirmed_incidence_prop\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)\n",
    "\n",
    "jhu_deaths_100 = covidcast.signal(\"jhu-csse\", \"deaths_incidence_prop\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)\n",
    "jhu_deaths_abs = covidcast.signal(\"jhu_case\", \"confirmed_incidence_num\",\n",
    "                        date(yr, strmon, strdt), date(yr, mon, enddt),\n",
    "                        geo_type=\"county\", geo_values=ctylist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 157,
   "metadata": {},
   "outputs": [],
   "source": [
    "frames = [move_home, move_fullw, move_partw, doc_visit, survey_c, survey_i,\n",
    "          hv, jhu_case, jhu_deaths_100, jhu_deaths_abs]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 158,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_symp = pd.concat(frames)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Policy dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Covid Policy data: https://github.com/OxCGRT\n",
    "# Citation: Thomas Hale, Tilbe Atav, Laura Hallas, Beatriz Kira, Toby Phillips, Anna Petherick, Annalena Pott. Variation in US states’ responses to COVID-19. Blavatnik School of Government.\n",
    "\n",
    "url = 'https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_US_states_temp.csv'\n",
    "dfpolicy = pd.read_csv(url, index_col=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfpolicy['Date'] = dfpolicy['Date'].astype(str)\n",
    "dfpolicy[\"date\"] = pd.to_datetime(dfpolicy['Date'], format='%Y%m%d')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
